### DISCUSSION TITLE
Can I somehow use your software to work with Hantek 6104BD?

### DISCUSSION THREAD

**Discussion by anatoltrush** [OpenHantek/OpenHantek6022#342] [Q&A] [answered]
Can I somehow use your software to work with Hantek 6104BD?

**Comment by Ho-Ro** [comment, marked as answer]
> It has 4 independent analog channels, 1GSa/s real-time sampling rate, ...

No chance, OpenHantek is only usable with the 6022 low cost HW (Cypress FX2LP based) - no buffer memory - direct stream of the samples via USB hi-speed.
